pkg/rotator/rotator.go

```diff
@@ -45,14 +45,17 @@ const (
 
 var crLog = logf.Log.WithName("cert-rotation")
 
-var vwhGVK = schema.GroupVersionKind{Group: "admissionregistration.k8s.io", Version: "v1beta1", Kind: "ValidatingWebhookConfiguration"}
-
 var crdGVK = schema.GroupVersionKind{Group: "apiextensions.k8s.io", Version: "v1beta1", Kind: "CustomResourceDefinition"}
 
 var _ manager.Runnable = &CertRotator{}
 
 var restartOnSecretRefresh = false
 
+type WebhookInfo struct {
+	Name types.NamespacedName
+	GVK  schema.GroupVersionKind
```

Comment:
I think it would be friendlier for the client to pass in an "enum" rather than a GVK, e.g.:

```
type WebhookType int
const (
  Validating WebhookType = iota
  Mutating
  CRDConversion
)
```

And then we can handle the detailed type mappings inside the rotator. Downside: when we switch from v1beta1 to v1 for the webhook APIs, clients won't be able to override. Upside: clients won't have to make that switch themselves.

Reply:
Need to keep the WebhookType as a struct, we still need the name.
Changing to:

Reply:
Yup I wasn't proposing getting rid of the struct, just changing "GVK" to "Type" as you've shown here. Thanks!

I don't feel strongly about "WebhookInfo" vs "CertifiableInfo." The former seemed more clear to me and everything we're working with _is_ a webhook - why is "Certifiable" better?

Reply:
Switched to Webhook